ga4gh/datamodel/variants.py

```diff
@@ -335,9 +353,8 @@ def _addFile(self, filename):
                 self._chromFileMap[chrom] = varFile
 
     def _convertGaCall(self, recordId, name, pysamCall, genotypeData):
-        call = protocol.GACall()
-        call.callSetId = recordId
-        call.callSetName = name
+        callSet = self.getCallSet(name)
+        call = callSet.getGaCall()
```

Comment:
I'm +0 on getting rid of the `getGaCall` method. However, we should have
```
call = protocol.GACall()
call.callSetId = callSet.getId()
call.callSetName = callSet.getSampleName()
call.sampleId = callSet.getSampleName()
```
These are protected variables, so we should never just reach in and use them (except from subclasses).